### PR TITLE
Show template [Compile] screen

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This shows template 'Compile' screen. 'template' means that this screen
would be used as a basic for the future final 'Compile' screen.

For https://github.com/Samsung/ONE-vscode/issues/331
Draft https://github.com/Samsung/ONE-vscode/pull/297

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
refer to animated gif of this PR: [one-vscode-pr.zip](https://github.com/Samsung/ONE-vscode/files/8385585/one-vscode-pr.zip)

File: package.json [hunk 1]
```diff
@@ -137,6 +137,7 @@
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
     "@vscode/test-electron": "^2.1.3",
+    "@vscode/webview-ui-toolkit": "^0.9.3",
```
Comment (hyunsik-yoon) [inline package.json:140]: I used `vscode webview-ui-toolkit` for vscode-specific tags in html.
- intro: https://code.visualstudio.com/blogs/2021/10/11/webview-ui-toolkit
- git: and this: https://microsoft.github.io/vscode-webview-ui-toolkit/?path=/story/library-badge--default

File: src/Compile/CompilePanel.ts [hunk 2]
```diff
@@ -154,12 +166,114 @@ export class CompilePanel {
...
         <h1>Compile</h1>
-        WIP
+        <div class='left-margin-24'>
+          <div class='compile-selections-container'>
+            <div class='text-row'>
+              <span class='two-column-text-left'>
+                    <span>&#8226;</span> Target Code:
+              </span>
+              <span class='two-column-text-right'>
+                <vscode-dropdown id='target-code'>
```
Comment (hyunsik-yoon) [inline src/Compile/CompilePanel.ts:183]: For `vscode-dropdown`, refer to https://github.com/Samsung/ONE-vscode/pull/431/files#r839132734

File: src/Compile/CompilePanel.ts [hunk 3]
```diff
@@ -154,12 +166,114 @@ export class CompilePanel {
       <head>
```
Comment (YongseopKim) [inline src/Compile/CompilePanel.ts:166]: in the future, this could be extracted to
```js
  return getHTML(toolkitUri, jsUri, cssUri);
```

**Review by YongseopKim — APPROVED** [review]
LGTM

File: src/Compile/CompilePanel.ts (hunk 3 above)
Comment (hyunsik-yoon) [reply src/Compile/CompilePanel.ts:166]: Yes. I left similar opinion as a `TODO` at L160. 😸

Comment (YongseopKim) [reply src/Compile/CompilePanel.ts:166]: 👍

**Review by jyoungyun — APPROVED** [review]
[hsyoon-1.zip](https://github.com/Samsung/ONE-vscode/files/8393823/hsyoon-1.zip)

LGTM

**Pull request merged**